extensions/objects/models/objects.py

```diff
@@ -889,3 +889,23 @@ class NumberWithUnits(BaseObject):
             'schema': Units.SCHEMA
         }]
     }
+
+
+class ListOfSetsOfHtmlStrings(BaseObject):
+    """List of sets of Html strings class."""
+
+    description = 'A list of sets of Html strings.'
+    edit_html_filename = 'list_of_sets_of_html_strings_editor'
+    edit_js_filename = 'ListOfSetsOfHtmlStringsEditor'
+    default_value = []
+
+    SCHEMA = {
+        'type': 'list',
+        'items': {
```

Comment:
I think there is already a `SetOfHtmlStrings` schema which can be reused here.

Reply:
Done.